doc/source/serve/index.md

```diff
@@ -27,16 +25,13 @@ Serve is particularly well suited for {ref}`serve-model-composition`, enabling y
 
 Serve is built on top of Ray, so it easily scales to many machines and offers flexible scheduling support such as fractional GPUs so you can share resources and serve many machine learning models at low cost.
 
-:::{tabbed} Installation
+## Quickstart
 
 Install Ray Serve and its dependencies:
 
 ```bash
 pip install "ray[serve]"
 ```
-:::
-
-:::{tabbed} Quickstart
 
 To run this example, install the following: ``pip install ray["serve"]``
```

Comment:
Looks redundant with the above? Also the quotes should be on the outside, `"ray[serve]"`
<img width="605" alt="Screen Shot 2022-08-10 at 1 12 20 PM" src="https://user-images.githubusercontent.com/5459654/184010987-567cebc4-807c-4cac-ad48-9411f15fc42d.png">